packages/react/src/utils.ts

```diff
@@ -15,3 +16,5 @@ export function assertRequiredSigner(signer: Signer): asserts signer is Required
 }
 
 export const DEFAULT_PAGINATED_QUERY_LIMIT = 10;
+
+export const DEFAULT_PUBLICATION_SORT_CRITERIA = PublicationSortCriteria.Latest;
```

Comment:
Is this used in more than one place or planned to?

Reply:
What is the aim with these consts? Are these exported by the package at all?

Reply:
To be honest is not worth it because the `sortCriteria` is only used in `ExplorePublications` query, i'll move it to the inline version that you suggested in the other comment